jbpm-bootstrap-kjar/pom.xml

```diff
@@ -0,0 +1,538 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>start.jbpm.org</groupId>
+    <artifactId>start.jbpm.org</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>jbpm-bootstrap-kjar</artifactId>  
```

Comment:
I think we should keep the original `groupId` to avoid complications in the generated project.